src/config.ts

```diff
@@ -0,0 +1,16 @@
+const SECONDS_IN_AN_HOUR = 60 * 60;
+
+// days * hours = 7 days
+const DEFAULT_EXPIRATION_HOURS = 7 * 24;
+
+const config = {
+  DEFAULT_DYNAMODB_TABLE: "stackJanitorTable",
```

Comment:
Can this be based off an environment variable, please?

Reply:
done.